index.html

```diff
@@ -2908,9 +2908,10 @@ <h3>Evidence</h3>
       <section>
         <h3>Zero-Knowledge Proofs</h3>
         <p class="issue" data-number="863">
-We plan to remove the normative requirements in this section if there are no
+We anticipate this section will undergo major revision for v2 of the data model,
+and plan to remove the normative requirements in this section if there are no
 active work items for securing <a>verifiable credentials</a> using zero
-knowledge proof mechanisms when we transition v2 of the data model to CR.
+knowledge proof mechanisms when we transition to v2 of the data model to CR.
```

Comment:
If this doesn't make sense to you, try reading both versions of the line aloud.
```suggestion
knowledge proof mechanisms when we transition v2 of the data model to CR.
```